File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
a, b = input().split()
a = int(a)
b = int(b)

c = int(input())

m = a*60 + b + c

a = int(m/60)
b = int(m%60)

if (a>=24):
  a = a - 24

print(a, b)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require('fs');
let input = fs.readFileSync('dev/stdin').toString().split('\n');

let hour = parseInt(input[0].split(' ')[0]);
let minute = parseInt(input[0].split(' ')[1]);

let needTime = parseInt(input[1]);

let total = (hour) * 60 + minute + needTime;

if(total >= 1440) total = total - 1440;

hour = parseInt(total / 60);
minute = total % 60;

console.log(`${hour} ${minute}`);